lya/data.py

```diff
@@ -22,7 +23,11 @@ def load_lya_data(filename=lya_data_path):
         K, matter power spectrum, and p(k) uncertainty arrays,
         encapsulated in Data object
     """
+    x, y, y_unc = np.loadtxt(filename, unpack=True)
+
+    data = Data(x, y, y_unc)
 
+    return data
```

Comment:
an `__init__` doesn't typically return anything

Reply:
this is not in __init__ 🤷🏻‍♂️ 

`load_lya_data` returns a data object. I thought this is what you had in mind according to the docstring.